x-pack/plugins/security_solution/common/detection_engine/utils.ts

```diff
@@ -17,6 +17,14 @@ export const hasNestedEntry = (entries: EntriesArray): boolean => {
   return found.length > 0;
 };
 
+export const hasEqlSequenceQuery = (ruleQuery: string | undefined): boolean => {
+  if (ruleQuery != null) {
+    const parsedQuery = ruleQuery.split(' ').filter((word) => word !== '' && word !== '\n');
```

Comment:
It still seems brittle for some reason, since there are four whitespace characters for EQL. `sequence\n[x]` may slip through.

`ruleQuery.trim().split(/[ \t\r\n]+/);` should do it the same way ANTLR does

Reply:
ah cool, i'll switch to that then